packages/hardhat-core/src/internal/hardhat-network/stack-traces/opcodes.ts

```diff
@@ -306,6 +306,10 @@ export enum Opcode {
   SELFDESTRUCT = 0xff,
 }
 
+export function opcodeName(opcode: number): string {
+  return opcodeNames[opcode];
```

Comment:
Typescript enums are two-way, so you can remove the `opcodeNames` array and do:

```suggestion
  return Opcode[opcode] ?? `<unrecognized opcode ${opcode}>`;
```

Reply:
🤦 

So much work 😭